lib/migration_helper.rb

```diff
@@ -0,0 +1,112 @@
+# This file aggregates helper methods which are used in various migrations
+module MigrationHelper
+
+  # Escapes a string literal so that it can be safely fed into PGSQL queries
+  # NOTE: with literals of this type prefix the string delimiters with E, e.g.:
+  # SELECT * FROM plant_populations WHERE plant_population_name = E'#{escaped_name}';
+  # (Because SQL standards are for losers... ;)
+  def escape(string)
```

Comment:
Deprecated in favor of `quote_value` but thanks all the same - seems no one on stackoverflow noticed it... ;)